config/pdk-settings.php

```diff
@@ -80,6 +81,11 @@
                     CarrierSettings::SHOW_DELIVERY_DAY                       => true,
                 ],
             ],
+            OrderSettings::ID   => [
+                SettingsManager::KEY_ALL => [
```

Comment:
dit kan niet, die key all is bedoeld voor collection settings, zoals carrier settings. nu gaat hij dit doen:

`orderSettings.*.statusOnLabelCreate`

+ dit moet ook voor de andere tristate opties, begin maar met de andere status update settings hier

En `-1` moet die tristate const zijn